### ISSUE TITLE
Youtube Subscription Importer No Longer Works

### ISSUE THREAD

**Issue by x4740N** [evanreilly/youtube-subscriptions-importer#3]
It Doesnt work, i did all steps correctly by exporting the subscriptions from the youtube subscription manager and importing the xml file onto your website

then i click on import and link my google account, then i click on import and it says its subscribed to people but when i check my you-tube account it does not do anything

**Comment by rudolphos** [comment]
Same here. It shows subscribed, but none of the subscriptions show up on youtube. Iguess they have changed their API or put some new limits in place.

**Comment by wapcrazut** [comment]
The web browser console drops Google API limits erros. So, if you use the website you will have erros for sure. The solution is to clone the project, setup your own API credentials with the Google Developers Console and edit `authentication.js`

**Comment by rudolphos** [comment]
@wapcrazut it works, but now it's limited to only 100 requests..
If I understand this correctly 1 request costs 100 quotas, which makes up 10k (daily query limit) for 100 subs.

After first 100, it's just `POST https://... 403`

**Comment by wapcrazut** [comment]
@rudolphos Yes, in theory is correct. According to the [Youtube Quota Calculator](https://developers.google.com/youtube/v3/determine_quota_cost) the `insert` method on `subscriptions` module cost about 53 quota.

**Comment by raj-khare** [comment]
I have made a tool similar to this but without the need for the XML file. https://raj-khare.github.io/yt-migrate/

[EDIT]
Correct url: https://github.com/raj-khare/yt-migrate

**Comment by mauroporras** [comment]
@raj-khare Not working, thanks.

**Comment by raj-khare** [comment]
@mauroporras Oh! Sorry. The correct url is this: https://github.com/raj-khare/yt-migrate

**Comment by mauroporras** [comment]
@raj-khare Thank you. I'm encountering this "External apps that request sensitive or restricted user data must first be verified by Google. This process may take 4-6 weeks."

Did you have to deal with the same?

**Comment by raj-khare** [comment]
@mauroporras Ya, I got the message, but you can proceed anyway by clicking on the advanced button in the OAUTH screen.

**Comment by Spinner65** [comment]
@raj-khare Thanks for alternative project.  I tried running it on OSX with it's oem python 2.7.   Got an Error.  I assume the code is for python 3 ?



**As for this youtube subscriptions importer project, @wapcrazut  Is correct. it is just an API error.  You can run the code on your own local webserver with your own APIs keys and it works as of the date of this post.**

**Comment by raj-khare** [comment]
@Spinner65 Yes, it requires python3. I have supplied my own API keys, but as stated quota will pretty much exhaust very soon. So you can either try again a day later (only remaining subscriptions will get transfer) or use your own keys.

**Comment by Spinner65** [comment]
> @Spinner65 Yes, it requires python3.

Thanks for the confirmation. I guess it is time to update the tools on my mac. :)

**Comment by mikewesten** [comment]
Nice. It worked. Thanks for that.

I had to use 'python**3** -m http.server 8080' instead of 'python -m http.server 8080' because of how my linux machine is set up. It still calls on Python 2.7 unless I issue a python3 command.
